src/Http.ts

```diff
@@ -1,5 +1,5 @@
 import Axios, { AxiosError, AxiosInstance } from 'axios'
-import qs from 'qs'
+import * as qs from 'qs'
```

Comment:
I think this is effectively the same as before.

Reply:
I added types package for qs and there is no default export